cinema_service/settings.py

```diff
@@ -133,9 +142,8 @@
 
 STATIC_URL = "static/"
 
+MEDIA_ROOT = "/files/media"
```

Comment:
The MEDIA_ROOT setting is currently set to an absolute path. It's recommended to build this path using the BASE_DIR setting to ensure it's always relative to your project directory.